src/Arcus.WebApi.Logging.Core/Correlation/HttpCorrelationInfoAccessor.cs

```diff
@@ -38,7 +39,7 @@ public CorrelationInfo GetCorrelationInfo()
         public void SetCorrelationInfo(CorrelationInfo correlationInfo)
```

Comment:
Maybe implement this method explicitly ?

Reply:
To hide it you mean? That wouldn't do much bc we use the interfaces in the application. In fact, we could maybe make this class `internal` bc the consumer doesn't interact with it this way.

Reply:
But it can't hurt to implement it and register the feature on HTTP Context again, if it's not much work?

Reply:
If folks want to use it then, they can

Reply:
Ok, implemented and tested.